charts/wildfly/README.md

```diff
@@ -31,6 +31,13 @@ build:
   uri: <git repository URL of your application>
 ```
 
+If the source repository is private, you must have a source secret created in the same namespace where you are building the application which allows athenticating to the repository.  Provide the name of the secret in the build section as follows:
```

Comment:
Could you please add the description of the `build.sourceSecret` to the list at https://github.com/wildfly/wildfly-charts/blob/main/charts/wildfly/README.md#building-the-application-image and links to https://docs.okd.io/latest/cicd/builds/creating-build-inputs.html#builds-manually-add-source-clone-secrets_creating-build-inputs ?

Reply:
WIll do.  BTW, the last column cuts off on my Brave Browser on Mac.  The link to the docs will be similar.  This should all work on OCP as well as OKD and perhaps other k8s systems, it is just that I am testing on OKD.

Reply:
BuildConfig is specific to OKD/OCP. This link should be fine.
More generally, we should link to k8s if the field relates to general kubernetes resources

Reply:
Done.  Do you want me to squash the commits?

Reply:
yes please

Reply:
done